contracts/epochs/FlowDKG.cdc

```diff
@@ -202,6 +202,19 @@ pub contract FlowDKG {
     /// The Admin resource provides the ability to begin and end voting for an epoch
     pub resource Admin {
 
+        /// Sets the optional safe DKG success threshold
+        pub fun setSafeSuccessThreshold(newThresholdPercentage: UFix64?) {
```

Comment:
why is it an optional?

Reply:
It is an optional so that the admin can clear it if it isn't needed any more